cookbook-front-end/src/App.tsx

```diff
@@ -127,17 +110,30 @@ const App = () => {
           }
         </Toolbar>
       </AppBar>
-      {data.currentPanel === Panels.browseRecipes && <Recipes/>}
-      {data.currentPanel === Panels.recipeDetail && <RecipeDetail recipeDetailId={data.recipeDetailId}/>}
-      {data.currentPanel === Panels.createRecipe && data.userIsLoggedIn && <CreateRecipe/>}
+
+      <Switch>
```

Comment:
This `<Switch>` sets up the main routing for our app. It's basically saying "If I see a certain path, load the specified component."

Reply:
makes sense. this is really really nice.